src/capstone_project_team_5/services/llm.py

```diff
@@ -60,11 +56,10 @@ def generate_bullet_points_from_analysis(
     - Mention only provided technologies; do not invent metrics or tools
     - Output lines prefixed with "- " only
     """
-    api_key = os.environ.get("GEMINI_API_KEY") or os.environ.get("GOOGLE_API_KEY")
-    if not api_key:
-        raise LLMError("Missing GEMINI_API_KEY/GOOGLE_API_KEY for Gemini")
-
-    model = os.environ.get("LLM_MODEL", "gemini-2.5-flash")
+    try:
+        llm_service = LLMService()
+    except RuntimeError as e:
```

Comment:
might wanna handle for other types exceptions as well


```suggestion
    except Exception as e:
```